x-pack/test_serverless/shared/services/index.ts

```diff
@@ -10,7 +10,8 @@ import { SvlCommonApiServiceProvider } from './svl_common_api';
 import { SvlReportingServiceProvider } from './svl_reporting';
 import { SvlUserManagerProvider } from './svl_user_manager';
 import { DataViewApiProvider } from './data_view_api';
-
+import { AlertingApiProvider } from './alerting_api';
```

Comment:
I moved these services from `test_serverless/api_integration` into `test_serverless/shared`, so it can be used from functional tests as well. I got inspired by this [PR](https://github.com/elastic/kibana/pull/183135)

Reply:
In the end as part of [this commit](https://github.com/elastic/kibana/pull/183113/commits/4d2182099a566d06c99cfd7085ef0ca292efc6f2), I moved alerting and api services under `test/api_integration` and then I added them under deployment agnostic services